esmvalcore/cmor/_fixes/fix.py

```diff
@@ -150,6 +154,7 @@ def get_fixes(project, dataset, mip, short_name):
         """
         cmor_table = CMOR_TABLES[project]
         vardef = cmor_table.get_variable(mip, short_name)
+        var_mapping = get_variable_details(project, dataset, mip, short_name)
```

Comment:
As discussed earlier, do you think it would make sense to pass these values in from the recipe instead of loading them here? That would have the advantage that the facets from the configuration file could be overwritten from the recipe.

They could be passed into the preprocessor functions `fix_file`, `fix_metadata`, and `fix_data` from the recipe here:
https://github.com/ESMValGroup/ESMValCore/blob/347d19d28c0b91891180a58a211ad7781c2d6999/esmvalcore/_recipe.py#L270-L278
Do check if that also works for fx files though, i.e. in the file [esmvalcore/preprocessor/_ancillary_vars.py](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/preprocessor/_ancillary_vars.py) these functions should also receive the correct arguments.